fact/io.py

```diff
@@ -0,0 +1,192 @@
+from os import path
+import pandas as pd
+import json
+import h5py
+import sys
+import logging
+import numpy as np
+from copy import copy
+
+__all__ = [
+    'write_data',
+    'to_native_byteorder',
+    'read_data',
+    'read_h5py',
+    'read_h5py_chunked',
+    'read_pandas_hdf5',
+    'check_extension',
+]
+
+log = logging.getLogger(__name__)
+
+
+allowed_extensions = ('.hdf', '.hdf5', '.h5', '.json', '.csv')
+native_byteorder = native_byteorder = {'little': '<', 'big': '>'}[sys.byteorder]
+
+
+def write_data(df, file_path, key='table'):
```

Comment:
No, totally not. It's already implemented in erna. This PR is about moving stuff from klaas here. I will open another one moving the stuff from erna here.